yt_dlp/extractor/digiteka.py

```diff
@@ -25,74 +25,83 @@ class DigitekaIE(InfoExtractor):
         )/(?P<id>[\d+a-z]+)'''
     _EMBED_REGEX = [r'<(?:iframe|script)[^>]+src=["\'](?P<url>(?:https?:)?//(?:www\.)?ultimedia\.com/deliver/(?:generic|musique)(?:/[^/]+)*/(?:src|article)/[\d+a-z]+)']
     _TESTS = [{
-        # news
-        'url': 'https://www.ultimedia.com/default/index/videogeneric/id/s8uk0r',
-        'md5': '276a0e49de58c7e85d32b057837952a2',
+        'url': 'https://www.ultimedia.com/default/index/videogeneric/id/3x5x55k',
         'info_dict': {
-            'id': 's8uk0r',
+            'id': '3x5x55k',
             'ext': 'mp4',
-            'title': 'Loi sur la fin de vie: le texte prévoit un renforcement des directives anticipées',
+            'title': 'Il est passionné de DS',
             'thumbnail': r're:^https?://.*\.jpg',
-            'duration': 74,
-            'upload_date': '20150317',
-            'timestamp': 1426604939,
-            'uploader_id': '3fszv',
+            'duration': 89,
+            'upload_date': '20251012',
+            'timestamp': 1760285363,
+            'uploader_id': '3pz33',
         },
-    }, {
-        # music
-        'url': 'https://www.ultimedia.com/default/index/videomusic/id/xvpfp8',
-        'md5': '2ea3513813cf230605c7e2ffe7eca61c',
-        'info_dict': {
-            'id': 'xvpfp8',
-            'ext': 'mp4',
-            'title': 'Two - C\'est La Vie (clip)',
-            'thumbnail': r're:^https?://.*\.jpg',
-            'duration': 233,
-            'upload_date': '20150224',
-            'timestamp': 1424760500,
-            'uploader_id': '3rfzk',
-        },
-    }, {
-        'url': 'https://www.digiteka.net/deliver/generic/iframe/mdtk/01637594/src/lqm3kl/zone/1/showtitle/1/autoplay/yes',
-        'only_matching': True,
+        'params': {'skip_download': True},
     }]
 
     def _real_extract(self, url):
         mobj = self._match_valid_url(url)
         video_id = mobj.group('id')
-        video_type = mobj.group('embed_type') or mobj.group('site_type')
-        if video_type == 'music':
-            video_type = 'musique'
 
-        deliver_info = self._download_json(
-            f'http://www.ultimedia.com/deliver/video?video={video_id}&topic={video_type}',
-            video_id)
+        IFRAME_MD_ID = '01836272'   # One static ID working for Ultimedia iframes
 
-        yt_id = deliver_info.get('yt_id')
-        if yt_id:
-            return self.url_result(yt_id, 'Youtube')
+        api_url = f'https://www.ultimedia.com/player/getConf/{IFRAME_MD_ID}/1/{video_id}'
+
+        conf_data = self._download_json(
+            api_url, video_id, note='Downloading player configuration')
```

Comment:
Can just inline this and make it fatal. If the API doesn't return a video, there is nothing useful we can do anyway
```suggestion
        video_info = self._download_json(
            f'https://www.ultimedia.com/player/getConf/{IFRAME_MD_ID}/1/{video_id}', video_id,
            note='Downloading player configuration')['video']
```